packages/core/src/new-api/types/deployment-loader.ts

```diff
@@ -8,9 +8,14 @@ import { Journal } from "./journal";
  */
 export interface DeploymentLoader {
   journal: Journal;
-  loadArtifact(storedArtifactPath: string): Promise<Artifact>;
-  storeArtifact(futureId: string, artifact: Artifact): Promise<string>;
-  storeBuildInfo(buildInfo: BuildInfo): Promise<string>;
+  loadArtifact(artifactFutureId: string): Promise<Artifact>;
+  storeArtifact(futureId: string, artifact: Artifact): Promise<void>;
```

Comment:
Maybe `storeUserProvidedArtifact` or something like that, to make it evident why it's different from the other one.

Reply:
Done